frontend/playground-frontend/src/components/Dashboard/Dashboard.js

```diff
@@ -126,63 +131,147 @@ const formatDate = (unixTime) => {
+                    <TableCell align="left" className="row-style">
+                      {formatDate(new Date(row.timestamp))}
+                    </TableCell>
+                    <TableCell align="left">
+                      <StatusDisplay
+                        statusType={row.status}
+                        status={`${row.progress.toFixed(2)}%`}
+                      />
+                    </TableCell>
+                    <TableCell align="left">
+                      <IconButton
+                        icon="download"
+                        accessibilityLabel={"Download"}
+                        size={"md"}
+                        onClick={async (e) => {
+                          e.event.stopPropagation();
+                          const response = await sendToBackend(
+                            "executionfiles",
+                            {
+                              exec_id: row.execution_id,
+                            }
+                          );
+                          console.log(response);
+                          //const thing = await fetch(response.dl_results, {
+                          //  mode: "cors",
+                          //});
+                          console.log([
+                            //response.dl_results,
+                            response.model_onnx,
+                            response.model_pt,
+                          ]);
+                          const zip = new JSZip();
+                          await Promise.all(
+                            [
+                              [response.dl_results, "dl_results.csv"],
+                              [
+                                response.model_onnx,
+                                "my_deep_learning_model.onnx",
+                              ],
+                              [response.model_pt, "model.pt"],
+                            ].map(([url, filename]) =>
+                              fetch(url, {
+                                mode: "cors",
+                              }).then((res) =>
+                                res.blob().then((blob) => {
+                                  zip.file(filename, blob);
+                                  /*let tempLink = document.createElement("a");
+                                  tempLink.href =
+                                    window.URL.createObjectURL(blob);
+                                  tempLink.setAttribute("download", filename);
+                                  tempLink.click();*/
+                                })
+                              )
+                            )
+                          );
+                          zip
+                            .generateAsync({ type: "blob" })
+                            .then((blob) => saveAs(blob, "results.zip"));
+                          //console.log(result);
+
+                          //set cors configuration for this bucket for all exec headers
```

Comment:
is this comment necessary here? @dwu359

Reply:
Oh yeah btw rn I set the cors configuration for s3 for all origins to be allowed, idk if that should be changed for prod. I'll delete those comments though

Reply:
I had to set cors config for presigned urls to work